src/main/java/com/connexta/poller/api/PollingService.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.poller.api;
+
+import java.util.concurrent.Future;
+
+/**
+ * This service accepts a PollingTask and returns a Future. The Future provides the results of the
```

Comment:
✏️ This class javadoc is just describing the method. The javadoc should explain the PollingService's responsibilities in the polling API.

Reply:
I'm struggling to come up with a good class comment.  
_A service that executes a poll task and a returns a value_
What can I said about it that isn't obvious?
I just removed the commend for now.

Reply:
Which piece of code will be calling the `poll` method and managing the Futures?

Reply:
That code hasn't been written yet. It won't be in this PR because it is tied to a different ticket. Well... I guess that is up for debate. You can read the other ticket -- it's in the sprint --and decide for yourself.

In my mind, I imagine only a single object, probably a @Service, talking to the poller code.

Reply:
I thought this ticket was for determining how polls are executed and the other ticket was the actual poll implementation.